bazel/pipeline.bzl

```diff
@@ -150,6 +153,7 @@ def sematic_pipeline(
             tag = "{}_{}".format(name, tag),
             format = "Docker",
             tags = tags,
+            insecure_repository = insecure_repository,
```

Comment:
Due to the fact that bazel workspace dependencies are not transient, some of our users may have an older `io_bazel_rules_docker` for which this option is not present. Can you instead make this variable only included as an arg here if `insecure_repository` is set to `True`?

Reply:
oh i see. im new to bazel... can i create like a kwargs and append it only if needed?

Reply:
@augray im new to bazel, do you want me (can i even do?) something like:

```
def sematic_pipeline(foo, **kwargs):
  ...
  push_kwargs = { ... }
  if "insecure_repository" in kwargs:
    push_kwargs["insecure_repository"] = kwargs["insecure_repository"]
  ...
  container_push(**kwargs)
```

Reply:
Ah yes. Sorry I missed the notification about this! I went ahead and did the suggested change. You can [merge this](https://github.com/v-pwais/sematic/pull/1) to accept my changes.

Reply:
thank you!!